app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -293,6 +293,9 @@ rf-project-editor .leaflet-tile {
   flex-direction: row;
   align-items: start;
   flex: 1;
+  & + .list-group-item-content {
```

Comment:
Fix menu - right now you have to click directly on the gradient for it to take hold.
We probably also want the currently selected gradient to be highlighted, and have a lighter highlight on the gradient you are currently hovering over.